src/components/App.js

```diff
@@ -432,10 +432,11 @@ class App extends Component {
         .filter((staffing) => { return this.state.companies[staffing.company] })
         .filter((staffing) => { return this.state.positions[staffing.position] })
       const projectToDisplay = this.state.globalProjects.filter((project) => {
-        return hasActiveCompanies(project.companies, this.state.companies)
+        return project.companies.some(company => {return(this.state.companies[company]== true)})
```

Comment:
what is the `== true` doing?

Reply:
the state.companies is a dictionary showing which companies have been selected by the user, so this is checking if any of the companies connected to a project are selected

Reply:
should it be === ?

Reply:
In general use === rather than == whenever you can, but here I think we don't even need it as this.state.companies[company] will be true by itself